crates/consensus/src/consensus/mod.rs

```diff
@@ -143,12 +143,16 @@ impl HeaderStage {
         &mut self,
         peer: &Peer,
         point: &Point,
-        raw_header: &RawBlock,
+        raw_header: &[u8],
     ) -> Result<(), WorkerError> {
-        let header: ConwayHeader = minicbor::decode(raw_header)
+        let header: babbage::MintedHeader<'_> = minicbor::decode(raw_header)
```

Comment:
Works as conway header share the same structure with babbage header, bu looks a bit confusing. Maybe `MintedHeader` should be made available in `conway` too?

Similarly, we probably need some abstraction here to deal with eventual future header changes introduced by future eras.

Reply:
That was the intention, but unfortunately, for some reasons, MintedHeader isn't publicly exposed in Pallas from the `conway` module :/ ... I think it's an oversight, but for now, I went with this.